.github/workflows/integration.yml

```diff
@@ -16,6 +12,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Verify branch
+        if: github.ref != 'refs/heads/main'
```

Comment:
Not sure but I think using `github.base_ref` might work better

Reply:
Yep, I was thinking the same, will try it, thanks for the suggestion.